src/sidebar/test/bootstrap.js

```diff
@@ -1,6 +1,10 @@
 // Expose the sinon assertions.
 sinon.assert.expose(assert, { prefix: null });
 
+// Patch any extra assert methods defined in assert-polyfills
+import patch from '../../test-util/assert-polyfills';
+patch(assert);
```

Comment:
@robertknight I'm just not sure about the names or if this was what you were expecting.

Reply:
Hi @LMS007 - This looks good to me except that the term _polyfill_ specifically refers to an implementation of a feature that is part of the documented behaviour or API of a system, but which is missing from the current implementation. Here we are adding extra methods to `assert` that are not part of its normal API.

Perhaps `assert-extensions` or `assert-methods`, with corresponding changes to comments in the assert module.